api/src/test/java/org/apache/iceberg/FileFormatTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class FileFormatTest {
+
+  @ParameterizedTest
+  @MethodSource
```

Comment:
I have a slight preference for FieldSource here because we don't actually need to dynamically create this.

@FieldSource("...")

Also we previously established in some other files that we wouldn't use implicit sources for tests and wanted to name them explicitly (even if it was the same as the implicit name). cc: @nastra

Reply:
Updated: [574c40f](https://github.com/apache/iceberg/pull/12301/commits/574c40fa3cd939bfb63f55287ebf65a664d8f591)